src/_posts/2020-05-19-promises-and-lies.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+asset-type: post
+name: promises-and-lies
+title: Promises and Lies
+date: 2020-05-19 10:00:00 +00:00
+author: Christopher Eyre
+description: Adding timeouts to promises in Javascript.
+image:
```

Comment:
This image doesn't require photo attribution. Can we please move the attribution to the bottom of the post if at all.